libqtile/resources/default_config.py

```diff
@@ -196,3 +196,10 @@
 # We choose LG3D to maximize irony: it is a 3D non-reparenting WM written in
 # java that happens to be on java's whitelist.
 wmname = "LG3D"
+
+
+# This hook causes qtile to reconfigure its screens when it receives randr
+# screen change events, for example when reconfiguring monitors with xrandr.
+@hook.subscribe.screen_change
+def reconfigure_screens(e):
+    qtile.reconfigure_screens
```

Comment:
This doubles as an example of how to use hooks, which is nice. :+1:

The call should probably be `qtile.reconfigure_screens(e)`, or is there some Python magic going on that I don't know about?
Also, would you mind calling the parameter `ev` or even `event`? For me personally, the biggest challenge with Python continues to be finding out what type an object has. More elaborate names help a little.

Reply:
Nope, that was me making a dumb typo when I wrapped `qtile.reconfigure_screens` inside the top-level function.

Thanks for the feedback, I've updated the commit.